dipy/reconst/ivim.py

```diff
@@ -513,7 +509,8 @@ def _leastsq(self, data, x0):
 
 class IvimModelVP(ReconstModel):
     def __init__(self, gtab, bounds=None, maxiter=10, xtol=1e-8):
-        r"""Initialize an IvimModelVP class.
+        r"""Initialize an IvimModelVP class :footcite:p:`LeBihan1988`,
+        :footcite:p:`Federau2012`, :footcite:p:`Fadnavis2019`.
```

Comment:
I have never tried putting footcite in a title, maybe it's problematic for some reason?

Reply:
Although this was assumed to be the case here:
https://github.com/dipy/dipy/pull/3310#issuecomment-2287664588

When building locally, this is rendered correctly:
https://github.com/dipy/dipy/pull/3316/files#diff-ba7f8dd2f3a9bd5b0c4da601efc4abc84940839ab91c88e0742ee216399ea9f1R299

So not sure what the problem is.